clients/src/main/java/org/apache/kafka/clients/admin/AlterReplicaDirOptions.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.clients.admin;
+
+import org.apache.kafka.common.annotation.InterfaceStability;
+import java.util.Map;
+
+
+/**
+ * Options for {@link KafkaAdminClient#alterReplicaDir(Map)}.
+ */
+@InterfaceStability.Evolving
+public class AlterReplicaDirOptions {
```

Comment:
How about this: I added an abstract class named `ConfigOptions`. This class will have a timeMs variable and `Integer timeoutMs()` API. This abstract class can be used to hold all future APIs that are shared among all (or most) "*ConfigOptions" classes.

On the other hand I also think it is OK to keep it as is given that there are is only one common API (i.e. timeMs()) shared among those classes.